src/promptflow/promptflow/_sdk/data/executable/README.md

```diff
@@ -0,0 +1,13 @@
+Exported entry file & its dependencies are located in the same folder. The structure is as below:
```

Comment:
Could we breakdown what are the 800M size required by streamlit?

Reply:
and pyarrow is also needed for streamlit (70MB)
![image](https://github.com/microsoft/promptflow/assets/26239730/459cf4dd-a342-4b8a-997e-d50d9bc0caea)

Reply:
offline synced. 

use clean venv and the final size is 80M in on-file mode now